apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsAnyOfControl.tsx

```diff
@@ -42,14 +42,22 @@ export function JsonFormsAnyOfControl({
     uischemas,
   )
 
-  const options = anyOfRenderInfos.map((anyOfRenderInfo) => {
-    const option = String(anyOfRenderInfo.label || anyOfRenderInfo.schema.const)
+  const options = anyOfRenderInfos
+    .map((anyOfRenderInfo) => {
+      if (anyOfRenderInfo.schema.format === "hidden") {
+        return null
+      }
 
-    return {
-      label: option.charAt(0).toUpperCase() + option.slice(1),
-      value: option,
-    }
-  })
+      const option = String(
+        anyOfRenderInfo.label || anyOfRenderInfo.schema.const,
+      )
+
+      return {
+        label: option.charAt(0).toUpperCase() + option.slice(1),
+        value: option,
+      }
+    })
+    .filter((option) => option !== null)
```

Comment:
[question] abit obtuse of me, but what necessitates this change? wouldn't the correct fix be on the priority so that `hidden` gets rendered above `anyOf` (or is the issue that a child schema has `format: hidden`)

Reply:
childschem has hidden! in this case the option itself `largeImage` and `floating` should not be shown - this is a workabout without having to use growthbook to hide it from other agencies